pkg/ops/dataServerOps_test.go

```diff
@@ -4,24 +4,31 @@
 package ops
 
 import (
+	"context"
 	"os"
 	"testing"
 	"time"
 
-	"github.com/tellor-io/telliot/pkg/tcontext"
+	"github.com/tellor-io/telliot/pkg/config"
+	"github.com/tellor-io/telliot/pkg/db"
+	"github.com/tellor-io/telliot/pkg/rpc"
 	"github.com/tellor-io/telliot/pkg/testutil"
 	"github.com/tellor-io/telliot/pkg/util"
 )
 
 func TestDataServerOps(t *testing.T) {
-
+	cfg := config.OpenTestConfig(t)
 	exitCh := make(chan os.Signal)
 	logSetup := util.SetupLogger()
 	logger := logSetup("debug")
-	ctx, _, cleanup := tcontext.CreateTestContext(t)
+	DB, cleanup := db.OpenTestDB(t)
 	defer t.Cleanup(cleanup)
+	client := rpc.NewMockClient()
+	proxy, err := db.OpenLocalProxy(DB)
+	testutil.Ok(t, err)
 
-	ops, err := CreateDataServerOps(ctx, logger, exitCh)
+	ctx := context.Background()
+	ops, err := CreateDataServerOps(ctx, logger, cfg, DB, &proxy, client, nil, nil, exitCh)
```

Comment:
```suggestion
	ops, err := CreateDataServerOps(context.Background(), logger, cfg, DB, &proxy, client, nil, nil, exitCh)
```

Reply:
I declared this one outside because ctx is used again on line 36:
`testutil.Ok(t, ds.Start(ctx, exitCh), "starting server")`